src/components/QRCode.tsx

```diff
@@ -19,6 +19,22 @@ type QRCodeProps = {
      */
     logo?: ImageSourcePropType;
 
+    /**
+     * If the logo to be displayed in the middle of the QR code is an SVG, then this prop needs to be used
+     * instead of standard `logo`
+     */
+    logoSVG?: React.FC<SvgProps>;
+
+    /**
+     * Background color to be used for logo, we set background only for SVG logos
+     */
+    logoBackgroundColor?: string;
```

Comment:
That name would be not entirely true I think.
In theory you could put a `.png` logo inside the QR code, and if this png would have an alpha opacity layer, then the background color would apply and be visible in behind this avatar.

Right now all our avatars in the App are png that have no opacity and take full width of the circle, so we cannot see any background, but it can be there.

I think correct naming would be:
 - `logoBackgroundColor`
 - `svgLogoFillColor` - because that one is indeed only for svg

Reply:
sounds good!